nibabel/spatialimages.py

```diff
@@ -312,24 +308,18 @@ def supported_np_types(obj):
     return set(supported)
 
 
-class ImageDataError(Exception):
-    pass
+class Header(SpatialHeader):
+  '''Alias for SpatialHeader; kept for backwards compatibility.'''
```

Comment:
Should this be marked deprecated? e.g.

    def __init__(self, *args, **kwargs):
        warnings.warn('Header is deprecated, use SpatialHeader', DeprecationWarning)
        SpatialHeader.__init__(self, *args, **kwargs)